assets/javascripts/omise-embedded-card.js

```diff
@@ -25,17 +25,11 @@ function showOmiseEmbeddedCardForm({
   }
   element.style.height = iframeElementHeight + 'px'
 
-  const localeMatching = {
```

Comment:
Since this is a distinct fix, can we implement in separate PR?

Reply:
Sure. It's better to do it in separate PR.